README.md

```diff
@@ -49,7 +57,12 @@ You can customize the map background if you provide your own [maplibre](https://
 MAPLIBRE_STYLE="https://your_tile_provider/style.json?key=42" npm run serve
 ```
 
-## Using liblrs
+### Python bindings
```

Comment:
You could also add a link to PyPI https://pypi.org/project/liblrs-python/

Reply:
done